src/Http/Requests/BaseExpoRequest.php

```
@@ -0,0 +1,21 @@
+<?php
+
+namespace NotificationChannels\ExpoPushNotifications\Http\Requests;
+
+use RonasIT\Support\Http\BaseRequest;
+use Illuminate\Contracts\Validation\Validator;
+use NotificationChannels\ExpoPushNotifications\Http\Resources\ExpoNotValidResource;
+use Symfony\Component\HttpKernel\Exception\UnprocessableEntityHttpException;
+
+class BaseExpoRequest extends BaseRequest
+{
+    protected function failedValidation(Validator $validator): ExpoNotValidResource
+    {
+        throw new UnprocessableEntityHttpException(
+            (new ExpoNotValidResource([
```

This looks very strange. In my opinion, a resource should not be used in the request. Resources are meant for controller responses. Can you throw an exception here without using a resource?